src/formats.txt

```diff
@@ -3,9 +3,9 @@ kg_failed	Key generation for %s: failed: %s (%s)
 kg_completed	Key generation for %s: completed in %d seconds. Reloading keys...
 kg_aborted_dup	Key generation for %s: aborted. Key generation for %s still in progress.
 kg_aborted_dir	Key generation for %s: aborted, failed to create directory %s: %s
-kg_mkdir	Created directory %s
-kg_pipe	Key generation for %s: error creating pipe: %s
-kg_fork	Key generation for %s: fork() error: %s
+kg_mkdir	created directory %s
+kg_pipe	        Key generation for %s: error creating pipe: %s
```

Comment:
Odd whitespace changes?

Reply:
Probably it just wasn't lining up right on my editor.

Reply:
I suspect @TingPing would prefer white space changes to be in a separate pull request, and not mixed with functional changes.